### PR TITLE
SYCL-Patatrack for GSoC 2022

### PR DESCRIPTION
**Pull request by tonydp03**
Proposal for the SYCL backend support with Alpaka in Patatrack Pixel Track reconstruction.

@felicepantaleo @waredjeb

### CODE REVIEW

File: _gsocproposals/2022/proposal_syclAlpakatoPatatrack.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * C++, Makefile, gcc, C++ smart pointers, C++ templates, bash shell scripting
+
+## Desirable Skills
+
+ * Experience with C++
+ * Experience with GNU Makefile
+
+## Mentors
+
+ * [Felice Pantaleo](mailto:clue-dev@cern.ch) (CERN)
```
Comment (hegner) [inline _gsocproposals/2022/proposal_syclAlpakatoPatatrack.md:44]: who is the main mentor? please indicate

File: gsoc/2022/mentors.md [hunk 2]
```diff
@@ -27,3 +27,6 @@ layout: plain
 * Mayank Sharma[Mayank Sharma](mailto:mayank.sharma@cern.ch) CERN
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) imperialcollege
 * Anna Zaborowska[Anna Zaborowska](mailto:anna.zaborowska@cern.ch) CERN
+* Felice Pantaleo [felice.pantaleo@cern.ch](mailto:felice.pantaleo@cern.ch) CERN
```
Comment (hegner) [inline gsoc/2022/mentors.md:30]: please insert mentors alphabetically.

**Review by hegner — CHANGES_REQUESTED** [review]

File: _gsocproposals/2022/proposal_syclAlpakatoPatatrack.md (hunk 1 above)
Comment (tonydp03) [reply _gsocproposals/2022/proposal_syclAlpakatoPatatrack.md:44]: Done

File: gsoc/2022/mentors.md (hunk 2 above)
Comment (tonydp03) [reply gsoc/2022/mentors.md]: Done

**Review by hegner — APPROVED** [review]

**Pull request merged**